queries/dune_v2/period_slippage.sql

```diff
@@ -103,12 +99,51 @@ other_transfers as (
       and (t.evt_tx_hash = lower('{{TxHash}}') or '{{TxHash}}' = '0x')
       and (solver_address = lower('{{SolverAddress}}') or '{{SolverAddress}}' = '0x')
 ),
-batch_transfers as (
+eth_transfers as (
+    select
+        bt.tx_hash,
+        from as sender,
+        to as receiver,
+        '0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee' as token,
```

Comment:
It does not really matter, but many people use: 0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE

Reply:
Dune uses all lower case addresses and (IMO) so should everyone else, always.